src/components/molecules/Card/Card.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import styled from 'styled-components'
+
+// A T O M S
+import { Icon, IconName, IconSize } from '../../atoms/Icon/Icon'
+
+export interface CardProps {
+  className?: string
+  footer?: React.ReactNode
+  icon?: IconName
+  iconSize?: IconSize
+  imageUrl?: string
+  imageMargin?: boolean
+  linkContainerMinHeight?: string
+  linkUrl?: string
+  style?: React.CSSProperties
+  title?: string
+}
+
+export const Card: React.FC<CardProps> = ({
+  children,
+  className,
```

Comment:
The optional `className` should have a default value `''` or you will see in the rendered DOM something like `class="card undefined"`.